MCFM-JHUGen/src/User/mdata.f

```diff
@@ -510,4 +510,3 @@ block data block_epinv
       data epinv2/1d3/
       end
 ************************************************************************
-
```

Comment:
Given that nothing is changed in this file now, it would be best to just restore the original. No need to have a change of whitespace for this file in its commit history.